Admin/AdminHelper.php

```diff
@@ -335,6 +334,8 @@ private function pathExists(PropertyAccessorInterface $propertyAccessor, $entity
             return true;
         } catch (NoSuchPropertyException $e) {
             return false;
+        } catch (UnexpectedTypeException $e) {
```

Comment:
@jordisala1991 can you please create a PR after we merged this one, where you remove this stuff, and explain WHY we remove it and refer to this PR?

Thank you 🍺